source/sass/patterns/organisms/content-header.scss

```diff
@@ -125,3 +126,42 @@
     @include rem(font-size, scale(4));
   }
 }
+
+.content-header__download_link {
+  @include inset-inline($end: 7%);
+  @include inset-block($start: 24px);
+  float: inline-end;
+  position: absolute;
+
+  @include mq($from: x-wide) {
+    @include inset-inline($end: 14%);
+    @include inset-block($start: 14px);
+  }
+
+  @include mq($from: x-wide) {
+    @include inset-inline($end: 42px);
+
+    @supports (inset-inline-end: 42px) {
+      @include inset-inline($end: 42px);
+    }
+  }
+
+  .content-header--image & {
```

Comment:
We haven't built the 'background image' variant yet, so don't need any `.content-header--image` selectors for now.